recipes/astunparse/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "astunparse" %}
+{% set version = "1.5.0" %}
+{% set hash_type = "sha256" %}
+{% set hash = "55df3c2a659d6cb6a9a9041c750a8232a9925523405a8dfeb891b92d45a589cd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - six >=1.6.1,<2.0
+    - wheel >=0.23.0,<1.0
```

Comment:
I think `wheel` isn't a real requirement of the package, but is instead part of the dev packaging requirements, so I think you can remove it.